ale_linters/swift/appleswiftformat.vim

```diff
@@ -0,0 +1,69 @@
+" Author: Klaas Pieter Annema <https://github.com/klaaspieter>
+" Description: Support for swift-format https://github.com/apple/swift-format
+
+let s:default_executable = 'swift-format'
+call ale#Set('swift_appleswiftformat_executable', s:default_executable)
```

Comment:
Even if longer I would prefer these names instead:

 - swift_apple_swift_format_executable
 - swift_apple_swift_format_use_swiftpm

Reply:
Indeed, I shoehorned it in the existing format 😅, but your proposal is better.